.flake8

```diff
@@ -0,0 +1,10 @@
+[flake8]
+# pep8 codes - http://pep8.readthedocs.io/en/release-1.7.x/intro.html#error-codes
+# E501 - line too long
+# E265 - block comment should start with '# '
+# E402 - module level import not at top of file
+# E999 - SyntaxError: EOL while scanning string literal
+# W504 - Line break occurred after a binary operator
+ignore = E265,E402,E999,W504
+max-line-length = 120
+exclude = .tox,.git,*/migrations/*,*/static/CACHE/*,docs,node_modules
```

Comment:
Nevermind on the dev-tools directory, I think we are fine with that included for now

Reply:
> I guess this is an extra safety measure in case there is a python file in one of the directories, correct? Do you know what the */migrations/* is in reference to?

The reason these are excluded because there's no reason for the linter to traverse the mentioned directories.
As for the "migrations" directory, I kept that in there for cases like the one you mentioned, where we might want a certain directory to be excluded.